### PR TITLE
1271 ccs telephone capture

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
CCS telephone capture

# What has changed
Handle CCS

# How to test?
mvn clean install and run ATs

# Links
https://trello.com/c/0T2BpkTE/1271-ccs-telephone-capture-5

### CODE REVIEW

File: src/test/java/uk/gov/ons/census/caseapisvc/service/UacQidServiceTest.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ public void createAndLinkUacQidReturnsALovelyObjectWhichIAmVeryFondOf() {
   public void calculateQuestionnaireTypeForHouseholdEngland() {
     // When
     int questionnaireType =
-        UacQidService.calculateQuestionnaireType("HH", "E1000", ADDRESS_LEVEL_UNIT);
+        UacQidService.calculateQuestionnaireType("HH", "E1000", "CENSUS", ADDRESS_LEVEL_UNIT);
```
Comment (ryangrundy7) [inline src/test/java/uk/gov/ons/census/caseapisvc/service/UacQidServiceTest.java]: This test is making `CENSUS` the address_level field

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍

**Review by LeoKHoward — APPROVED** [review]
Works for me

**Review by NickGrantX — CHANGES_REQUESTED** [review]
won't build for me

**Review by NickGrantX — APPROVED** [review]
Builds for me now 👍

**Pull request merged**